coral-dbt/build.gradle

```diff
@@ -14,6 +14,8 @@ spotless {
 task runCoralDbtTests {
   // Install Python dependencies
   def installModulesProcessBuilder = new ProcessBuilder([
+    'python3',
```

Comment:
does it mean the build will fail with python2?
if so, could you also update the Projetc's README to include a dependency on Python3?